x-pack/test_serverless/api_integration/test_suites/observability/apm_api_integration/common/apm_api_supertest.ts

```diff
@@ -105,7 +105,7 @@ async function getApmApiClient({
 }) {
   const url = format({
     ...kibanaServer,
-    auth: `${username}:${APM_TEST_PASSWORD}`,
+    auth: `${username}:${kbnTestConfig.getUrlParts().password}`,
```

Comment:
We are using `APM_TEST_PASSWORD` in a couple of other places. Can you replace them as well?

https://github.com/elastic/kibana/blob/c8083dd6767a88f15379531319b5636d20e79451/x-pack/test/apm_api_integration/common/bootstrap_apm_synthtrace.ts#L40

https://github.com/elastic/kibana/blob/c8083dd6767a88f15379531319b5636d20e79451/x-pack/test/apm_api_integration/common/config.ts#L43

https://github.com/elastic/kibana/blob/c8083dd6767a88f15379531319b5636d20e79451/x-pack/test/apm_api_integration/common/registry.ts#L158

Reply:
@sqren That's actually a good point. I completely got rid of the APM_TEST_PASSWORD

Reply:
Thanks @achyutjhunjhunwala !